src/modules/routing/messages/RoutingMessageType.ts

```diff
@@ -1,6 +1,7 @@
 export enum RoutingMessageType {
   // TODO: add other messages from mediator coordination protocol
   KeylistUpdate = 'https://didcomm.org/coordinatemediation/1.0/keylist_update',
```

Comment:
This is still snake_case. the PR was recently updated to use kebab-case

```suggestion
  KeylistUpdate = 'https://didcomm.org/coordinatemediation/1.0/keylist-update',
```

Reply:
Updated.